website/docs/users-sources/access-control/permissions.md

```diff
@@ -13,38 +13,39 @@ For instructions to add, remove, and manage permissions, refer to [Manage Permis
 
 ## Fundamentals of authentik permissions
 
+A [role](../roles/index.md) is a collection of permissions. A user or a group may have any number of roles. A user has a certain permission if they have a role that has that permission, or if they are part of a group (either directly or indirectly) which has a role that has that permission.
+
+- Example 1 (no group): Judith has the role "RADIUS", which has every permission for RADIUS providers and property mappings. Then, Judith has permission to add/view/change/delete RADIUS providers or RADIUS property mappings.
+- Example 2 (direct group): Marie is part of the group "Auditors". That group has the role "Event Log manager", which in turn has the permissions "Can view Event", "Can change Event", and "Can delete Event". Then, Marie has permissions to view, change, or delete Events.
+- Example 3 (indirect group): Elaine is part of the group "Accounting", which has a parent group "Back office", which has a parent group "Employees". The group "Employees" has the role "Read-only, which has view permissions on all object types. Then, Elaine has the permission to view any object in authentik through indirect membership of the "Employees" group.
+
+:::info
+From 2025.12, authentik's access control is fully role-based. Before 2025.12, Admins could assign permissions to individual [users](../user/index.mdx). To mimic this behavior of "User permissions", an Admin can
+:::
+
 There are two main types of permissions in authentik:
 
 - [**Global permissions**](#global-permissions)
 - [**Object permissions**](#object-permissions)
 
-Additionally, authentik employs _initial permissions_ to streamline the process of granting object-level permissions when an object (user or role) is created. This feature enables an Admin to proactively assign specific rights to a user for object creation, as well as for viewing and managing those objects and other objects created by individuals in the same role. For more details, refer to [Initial permissions](./initial_permissions.mdx).
+Additionally, authentik employs _initial permissions_ to streamline the process of granting object-level permissions when an object (user or role) is created. This feature automatically adds permissions for newly created objects to the role that created them. For more details, refer to [Initial permissions](./initial_permissions.mdx).
```

Comment:
If an admin still has to create the IP, as show in our How To steps, then it is misleading to not say that this has to be manually implemented, and to only say that the permissions are automatically assigned. I think we can say both, something like:

"When an Admin implements initial permissions, this feature automatically adds permissions for newly created objects to the role (and thus users with that role) that created them."